aws/vars.tf

```diff
@@ -30,6 +35,17 @@ variable "enable_monitoring" {
   default     = true
 }
 
+variable "clients" {
+  description = "Number of kafka client hosts to set up, if any."
+  type        = number
+  default     = 0
+}
+
+variable "client_distro" {
```

Comment:
Is there a reason to have a different distro for the client vs the rest of the cluster?

Reply:
In the case that some tools/tests are easier to build on a different (newer) distro. Default is the same distro though.